astropy/coordinates/builtin_frames/utils.py

```diff
@@ -19,12 +19,12 @@
 # The UTC time scale is not properly defined prior to 1960, so Time('B1950',
 # scale='utc') will emit a warning. Instead, we use Time('B1950', scale='tai')
 # which is equivalent, but does not emit a warning.
-EQUINOX_J2000 = Time('J2000', scale='utc')
+EQUINOX_J2000 = Time('J2000', scale='tt')
 EQUINOX_B1950 = Time('B1950', scale='tai')
```

Comment:
#8600 has `B1950` in TT not TAI.  Those are the same for B1950, right?  But in any case it would be good to be consistent with TT everywhere as the default for epoch times.

Reply:
Hmm.  I was thinking to leave `'tai'` here because it's not really at all clear what the "right" answer is since TT was adopted more recently than 1950.  But now that I say that, the same is true of TAI, so I guess consistency might really be more important here.

Reply:
I don't have a strong basis for an opinion here, but see that you just changed it, so that's fine.